test/ode/ode_convergence_tests.jl

```diff
@@ -13,128 +13,133 @@ testTol = 0.2
 
 for i = 1:2
   prob = probArr[i]
-  sim = test_convergence(dts,prob,Euler())
```

Comment:
This looks like a mistake.

Reply:
yeah I forgot to uncomment. just pushed changes